wpilib/wpilib/driverstation.py

```diff
@@ -118,6 +118,7 @@ def __init__(self):
         if not hasattr(DriverStation, 'instance') or DriverStation.instance is not None:
             raise ValueError("Do not create DriverStation instances, use DriverStation.getInstance() instead")
 
+        hal.initialize(500, 0)
```

Comment:
Erm. If you're actually going to do this, you should also remove the corresponding call in the hal package itself. But I'm not sure we should do this.

Reply:
I would have to look closer, but I'm 98% sure we would not want to do this here. Probably best to comment it out and add a comment saying that we're intentionally leaving it out.

Reply:
Thad mentioned upstream added it here to help with unit testing, since Java doesn't really have an equivalent of ensuring some code gets executed on first import. Of course, we ensure the HAL is initialised whenever it's imported, so this shouldn't be a concern for us.